plugins/com.google.cloud.tools.eclipse.appengine.whitelist.test/pom.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.google.cloud.tools.eclipse</groupId>
+    <artifactId>trunk</artifactId>
+    <version>94.0.0-SNAPSHOT</version>
+    <relativePath>../../</relativePath>
+  </parent>
+  <groupId>com.google.cloud.tools.eclipse</groupId>
+  <artifactId>com.google.cloud.tools.eclipse.appengine.whitelist.test</artifactId>
+  <packaging>eclipse-test-plugin</packaging>
+
+  <build>
+    <sourceDirectory>src</sourceDirectory>
+    <plugins>
+      <plugin>
+        <groupId>org.eclipse.tycho</groupId>
+        <artifactId>tycho-surefire-plugin</artifactId>
+        <version>${tycho.version}</version>
```

Comment:
Hmm, I thought I found it there; I must have looked at GPE's pom.xml instead.  We should definitely add `tycho-surefire-plugin` to the top pom's <pluginManagement> section — updating to the next Tycho will be problematic.